src/repositories/QueueRepository/__tests__/QueueRepository.int.spec.ts

```diff
@@ -208,4 +209,34 @@ describe("Queue Repository tests", () => {
     expect(DateTime.fromJSDate(playerInDb?.queueTime!).toISO()).toEqual(mockBallChaser.queueTime.toISO());
     expect(playerInDb?.team).toBeNull();
   });
+  it("check if player is in queue", async () => {
+    const mockBallChaser = BallChaserQueueBuilder.single();
+    await manuallyAddBallChaserToQueue(mockBallChaser);
+
+    const playerInQueue = await QueueRepository.isPlayerInQueue(mockBallChaser.id);
+
+    const playerInDb = await prisma.queue.count({
+      where: {
+        playerId: mockBallChaser.id,
+      },
+    });
+
+    expect(playerInDb).toEqual(1);
+    expect(playerInQueue).toEqual(true);
```

Comment:
Do we need to do this assertion? What functionality does this verify?
```
expect(playerInDb).toEqual(1);
```

Reply:
Nothing, fixed in next commit.